packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -318,47 +197,66 @@ function check_lock_file()
     return true
 end
 
+-- Remove lock file
 function remove_lock_file()
-    execute("rm /etc/first_run")
```

Comment:
if we have a function to check it, would also create a function to remove it so it is api symetrical

Reply:
I don't understand what you're asking for.

Reply:
my mistake